lib/app/item.ex

```diff
@@ -10,11 +10,11 @@ defmodule App.Item do
   @derive {Jason.Encoder,
            except: [:__meta__, :__struct__, :timer, :inserted_at, :updated_at]}
   schema "items" do
-    field :person_id, :integer
-    field :status, :integer
-    field :text, :string
+    field(:person_id, :integer)
```

Comment:
what formatter are you using that adds these parenthesis?

Reply:
I think it's this extension from VS Code:
<img width="298" alt="image" src="https://github.com/dwyl/mvp/assets/2154092/6220c5a9-50ee-4b31-8b79-a1e6d37fa57b">

But I just reverted it back to not change these.